app/classifier/tasks/dropdown/index.cjsx

```diff
@@ -100,7 +105,7 @@ module.exports = React.createClass
       @handleFocus()
 
   handleFocus: ->
-    @refs['select-0'].getInputNode().focus()
+    @refs['select-0'].focus()
```

Comment:
With back-to-back single dropdowns I get the following error: `TypeError: this.refs.select-0.focus is not a function(…)`. Here's a demo workflow: `projects/markb-panoptes/focus-testing-project/classify?reload=0&workflow=2738`. I've tried just using the Select.Creatable component only, confirmed `ref="select-0"` is indeed in the Select component, not sure what's going on. Maybe it's trying to focus to that component ref before the component has rendered?